app/services/jobs/return-logs/process-return-logs.service.js

```diff
@@ -1,31 +1,39 @@
 'use strict'
 
 /**
- * Creates the return logs for the next cycle
- * @module CreateReturnLogsService
+ * Process the return logs for the next cycle
+ * @module ProcessReturnLogsService
  */
 
 const { calculateAndLogTimeTaken, currentTimeInNanoseconds } = require('../../../lib/general.lib.js')
-const CreateReturnLogsService = require('./create-return-logs.service.js')
 const FetchReturnLogsService = require('./fetch-return-logs.service.js')
+const ReturnLogModel = require('../../../models/return-log.model.js')
 
 /**
  * Creates the return logs for the next cycle
  */
-async function go (isSummer, licenceReference) {
+async function go (cycle, licenceReference) {
   try {
+    const isSummer = cycle === 'summer'
     const startTime = currentTimeInNanoseconds()
     const returnLogs = await FetchReturnLogsService.go(isSummer, licenceReference)
 
     if (returnLogs.length > 0) {
-      await CreateReturnLogsService.go(returnLogs)
+      await _createReturnLogs(returnLogs)
     }
-    calculateAndLogTimeTaken(startTime, 'Process return logs complete', { isSummer, licenceReference })
+    calculateAndLogTimeTaken(startTime, 'Create return logs complete', { isSummer, licenceReference })
```

Comment:
To be consistent it should be the following message. And ideally we'd log the the cycle so we can see what was requested rather than what was inferred.

```javascript
    calculateAndLogTimeTaken(startTime, 'Create return logs job complete', { cycle, licenceReference })
```